tools/project_manager/show_good_word_suggestions_detail.php

```diff
@@ -45,109 +46,100 @@
     $userSettings->set_value("show_good_words_layout", $layout);
 }
 
-// $frame determines which frame we're operating from
-// 'master' - we're the master frame
-//   'left' - we're the left frame with the text
-//  'right' - we're the right frame for the image
-$frame = get_enumerated_param($_GET, 'frame', 'master', array('master', 'left', 'right'));
-
-if($frame=="master") {
-    slim_header_frameset(_("Suggestion Detail"));
-    if($layout == LAYOUT_HORIZ) $frameSpec='rows="30%,70%"';
-    else $frameSpec='cols="30%,70%"';
-?>
-<frameset <?php echo $frameSpec; ?>>
-<frame name="worddetailframe" src="show_good_word_suggestions_detail.php?projectid=<?php echo $projectid; ?>&amp;word=<?php echo $encWord; ?>&amp;timeCutoff=<?php echo $timeCutoff; ?>&amp;frame=left">
-<frame name="imageframe" src="show_good_word_suggestions_detail.php?projectid=<?php echo $projectid; ?>&amp;word=<?php echo $encWord; ?>&amp;timeCutoff=<?php echo $timeCutoff; ?>&amp;frame=right">
-</frameset>
-<noframes>
-<?php echo _("Your browser currently does not display frames!"); ?>
-</noframes>
-<?php
-    exit;
+$header_args = [
+  "js_files" => [
+    "$code_url/scripts/splitControl.js",
+    "$code_url/scripts/page_browse.js",
+    "./show_good_word_suggestions_detail.js",
+  ],
+  "js_data" => get_page_data_js($projectid, $imagefile, 'OCR', $error_message) . get_proofreading_interface_data_js() . "
+        var show_good_word_suggestions_details = {
```

Comment:
It would be better to just `json_encode()` a PHP associative array so that we don't have to worry about any escaping here.